utils/python/tests/scripts_regression_tests

```diff
@@ -916,6 +916,95 @@ class TestSingleSubmit(TestCreateTestCommon):
                          msg="COMMAND SHOULD HAVE WORKED\nwait_for_tests output:\n%s\n\nerrput:\n%s\n\ncode: %d" % (output, errput, stat))
 
 ###############################################################################
+class XMLQUERY(unittest.TestCase):
+# Testing command line scripts    
+###############################################################################
+    
+    def setUp(self):
+        # Create case directory
+        self._testroot = MACHINE.get_value("CESMSCRATCHROOT")  # "/tmp/" 
+        self._testdirs = []
+        self._do_teardown = []
```

Comment:
I mean the self._do_teardown member specifically. It doesn't seem to do anything.